Druid/DRCogitDispatchTableBuilder.class.st

```diff
@@ -0,0 +1,82 @@
+"
+Uses a Pharo Code Generator to build methods conforming to the Cogit JIT dispatch table specification. Currently these can be found browsing :
+
+- #primitiveTable
+- #initializePrimitiveTable
+
+"
+Class {
+	#name : #DRCogitDispatchTableBuilder,
```

Comment:
Maybe in the next version this should be called CogitPrimitiveDispatchTableBuilder?